kqueen/engines/heat.py

```diff
@@ -0,0 +1,178 @@
+from collections import namedtuple
+import importlib
+import yaml
+import logging
+import sys
+import os
+
+from kqueen.config import current_config
+import openstack
+
+from .base import BaseEngine
+
+logger = logging.getLogger('kqueen_api')
+config = current_config()
+
+
+STATE_MAP = {
+    'IN_PROGRESS': config.get('CLUSTER_PROVISIONING_STATE'),
+    'COMPLETE': config.get('CLUSTER_OK_STATE'),
+}
+
+
+class Heat:
+    templates = {
+        "default": "kqueen.engines.resources.heat.default",
+        "empty": "kqueen.engines.resources.heat.empty",
+    }
+
+    def __init__(self, **kwargs):
+        self.connection = openstack.connection.Connection(**kwargs)
+
+    def _get_template_and_files(self, name):
```

Comment:
I don't like that name, one method -> one purpose.
So please, separate them and add descriptions of what files we are getting (for what purpuse)
Does template can change? Path to the template can be stored as a constant (with file name) or set up from the config file

Reply:
or maybe clear renaming can resolve it? i mean to describe 1 action item in naming. for ex get_template_sources or smth similar